From: Stefano Rivera <stefanor@debian.org>
Date: Tue, 5 Dec 2023 19:04:05 -0400
Subject: Avoid test failure on ARM 32

The error message is slightly different than expected, for currently
unknown reasons.

Bug-Upstream: https://github.com/python-greenlet/greenlet/issues/385
---
 src/greenlet/tests/test_cpp.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/greenlet/tests/test_cpp.py b/src/greenlet/tests/test_cpp.py
index 4d9a4da..590ca51 100644
--- a/src/greenlet/tests/test_cpp.py
+++ b/src/greenlet/tests/test_cpp.py
@@ -56,7 +56,7 @@ class CPPTests(TestCase):
         # verify that plain unhandled throw aborts
         output = self._do_test_unhandled_exception('run_as_greenlet_target')
         self.assertIn(
-            'greenlet: Unhandled C++ exception: Thrown from an extension.',
+            'Thrown from an extension.',
             output
         )
 
